docs/guide/developing/resources/create.md

```diff
@@ -0,0 +1,110 @@
+# Create
+
+## Description
+
+Device with collection resource allows dynamic creation of resources of selected type.
+
+## Guide
+
+To develop your own device you can examine the example in [cloud_server](https://github.com/iotivity/iotivity-lite/blob/master/apps/cloud_server.c). Lets examine the example to identify the necessary steps that allow a device to dynamically create resources.
+
+1. Create a collection resource
+```C
+  oc_resource_bind_resource_type(col, "oic.wk.col");
+```
+2. Determine which resource types can populate the collection
+```C
+  oc_collection_add_supported_rt(col, "oic.r.switch.binary");
+  oc_collection_add_mandatory_rt(col, "oic.r.switch.binary");
+```
+3. Enable create operation on the collection resource
+```C
+oc_resource_bind_resource_interface(col, OC_IF_CREATE);
+```
+4. Define constructor and destructor functions for collection item instances
+```C
+oc_collections_add_rt_factory("oic.r.switch.binary", get_switch_instance, free_switch_instance);
+```
+5. Compile and link
+
+To enable create operation in iotivity-light library compile with CREATE=1 option.
+
+```make
+make cloud_server CLOUD=1 SECURE=0 CREATE=1
+```
+
+### Create a resource
+
+When you have a cloud backend and cloud_server binary running. You can use cloud client to create a resource.
+
+#### Using go grpc client
+
+Go [grpc client](https://github.com/plgd-dev/cloud/tree/master/bundle/client/grpc) is a simple tool that supports several useful commands we can combine to create a resource.
+
+1. Use the get command to get the id and href of a device with collection
+```bash
+./grpc -get
```

Comment:
Which resource is retrieved here (no href specified)?

Reply:
this call retrieves al resources at the backend, ofcourse you can specify deviceId and href to filter output.

Reply:
-get returns all devices, but maybe I can filter it somehow so only the collection is returned? @jkralik

Reply:
-get returns all resources; we will rename the method later on but to make it clear, can you add a commend right behind this call ?
`./grpc -get //retrieves all resources of all devices`